docs/src/app/components/markdown-element.jsx

```diff
@@ -0,0 +1,48 @@
+const React = require('react');
+let {Paper} = require('material-ui');
+require('./mui-github-markdown.css');
+import marked from 'marked';
+
+const MarkdownElement = React.createClass({
+  propTypes: {
+    text: React.PropTypes.string.isRequired,
+  },
+  getDefaultProps() {
+    return {
+      text: '',
+    };
+  },
+
+
+  componentWillMount() {
+    marked.setOptions({
+      gfm: true,
+      tables: true,
+      breaks: false,
+      pedantic: false,
+      sanitize: false,
+      smartLists: true,
+      smartypants: false,
+      highlight: function(code, lang) {
+        return require('highlight.js').highlight(lang, code).value;
+      },
+    });
+  },
+
+  render() {
+    const {text} = this.props;
+    const html = marked(text || '');
+
+/* eslint-disable */
```

Comment:
To allow html-tags within markdown (e.g. for definition lists) `dangerouslySetInnerHTML` is needed which is prevented in `.eslint`L74 no-danger.